static/scripts/datetime/datetime.js

```diff
@@ -53,11 +58,16 @@ const inputRange = ({
 	];
 };
 
+const dateStringToMoment = (date) => {
+	return (getCookie('USER_LANG') === 'de') ? moment(date, INTER_DATE_FORMAT.de) : moment(date, INTER_DATE_FORMAT.en);
```

Comment:
It should be like this. The correct locale is retrieved automatically.
```suggestion
const dateStringToMoment = (date, format = DATETIME_FORMAT.date()) => moment(date, format);
```